UPGRADING.md

```diff
@@ -66,7 +87,7 @@ are:
 - `blockchain`
 - `evidence`
 
-Accordingly, the space `node` package was changed to reduce access to
+Accordingly, the `node` package was changed to reduce access to
```

Comment:
lol nice catch

Reply:
it was my mistake in the first place! :p